evennia/contrib/worldloader/example_tutorial_world/worlddata/csv/world_details.csv

```diff
@@ -0,0 +1,54 @@
+key,name,location,desc
```

Comment:
I presume this is to be used with some sort of external editor? CSV text is cool for transfering data etc but it's after all not easy to read nor write by humans:  especially since the order of elements in each row is important and missing any one comma will mess up all that comes after it ... If so, this should probably be made more explicitly clear.